src/mkdocs_autorefs/plugin.py

```diff
@@ -14,6 +14,7 @@
 import functools
 import logging
 from typing import Callable, Dict, Optional, Sequence
+from urllib.parse import urlparse
```

Comment:
```suggestion
from urllib.parse import urlsplit
```
* https://github.com/mkdocs/mkdocs/commit/fd0e9dedd27e4cb628ab98ff2723165110b38ed2